frontend/js/src/settings/link-listens/LinkListens.tsx

```diff
@@ -198,10 +203,26 @@ export default function LinkListensPage() {
         <title>Link with MusicBrainz</title>
       </Helmet>
       <h2 className="page-title">Link with MusicBrainz</h2>
+      <ReactTooltip id="matching-tooltip" multiline>
+        We try our best to match every listen with a MusicBrainz recording in
+        order to enrich the information about the track, album and artist.
```

Comment:
Me with the red pen, as always... suggestions:
"We automatically match listens with MusicBrainz recordings when possible, which provides data like track, album, artist, cover art, and more."

Reply:
This is specifically why I request a review from you !